students/LuisGonzalez/session07/html_render.py

```diff
@@ -0,0 +1,112 @@
+
+class Element(object):
+    tag = "html"
+    indent = 1
+    def __init__(self, content = None, **kwargs):
+
+        self.content = content
+
+        if len(kwargs) > 0:
+            self.extraAttributes = str(kwargs).strip('{}')
+            self.extraAttributes = self.extraAttributes.replace("'", "", 2)
+            self.extraAttributes = " " + self.extraAttributes.replace("'", '"')
+        else:
+            self.extraAttributes = None
+        if self.content is None:
```

Comment:
For ease of reading, it's good to separate these by a line so it's clear they're separate. It also doesn't hurt to comment what some of these methods do... :)